settings/collect.ini

```diff
@@ -53,6 +53,13 @@ CollectAnonymousData=enabled
 CollectAnonymousDataList[]
 CollectAnonymousDataList[poll]=enabled
 CollectAnonymousDataAttribute=collection_anonymous
+# Are collected data sensitive? If true then user will be able to view collected
+# data only when previously submit a form
+CollectSensitiveData=true
+# Same as CollectSensitiveData but is a list with of IC types and
+# their override settings, if specified it will override default setting
+CollectSensitiveDataList[]
+CollectSensitiveDataList[poll]=false
```

Comment:
Nitpick maybe, but this line should be commented, so that we don't accidentally open up access to some IC type that was supposed to be closed. But: Not sure if `poll` is supplied by us. If it is, the setting should probably stay as is, so the functionality isn't blocked.